local-dev/clair/config.yaml

```
@@ -13,6 +13,12 @@ matcher:
   connstring: host=clair-db port=5432 user=clair dbname=clair sslmode=disable
   max_conn_pool: 100
   migrations: true
+remote_matchers:
+  sets:
+    - crda
+  config:
+    crda:
+      url: https://f8a-analytics-preview-2445582058137.production.gw.apicast.io/?user_key=3e42fa66f65124e6b1266a23431e3d08
```

Can we follow the below config format?
```suggestion
remote_matchers:
  sets:
    - crda
  config:
    crda:
      ecosystems:
          - 
           url: https://f8a-analytics-preview-2445582058137.production.gw.apicast.io/?user_key=3e42fa66f65124e6b1266a23431e3d08
           repositoryName: pypi
           requestConcurrencyLimit: 20
          -
           url: https://f8a-analytics-preview-2445582058137.production.gw.apicast.io/api/v2/component-analyses/%s/%s?user_key=3e42fa66f65124e6b1266a23431e3d08
           repositoryName: maven
           requestConcurrencyLimit: 30
```

---

It is not impossible but...making calls to different apis from inside remote_matcher.I dont know if its necessary.Why cant we use stack analysis api?